lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -55,31 +59,34 @@ class DetailsRenderer {
         // @ts-ignore - TODO(bckenny): Fix type hierarchy
         return this._renderTable(/** @type {TableDetailsJSON} */ (details));
       case 'code':
-        return this._renderCode(details);
+        return this._renderCode(/** @type {StringDetailsJSON} */ (details));
```

Comment:
if we're killing string details json elsewhere should we be moving this to it?

Reply:
> if we're killing string details json elsewhere should we be moving this to it?

I can leave it as `DetailsJSON`? Basically I need to tell it that it's not an `OpportunityDetails`, and until all the details types are known to the compiler with `type` defined it isn't narrowing it down here

Reply:
Oh I meant elsewhere it seems like we're deleting `StringDetailsJSON` and going with literal `{text: string}` typedefs, so should we just do that instead